src/visit.ts

```diff
@@ -517,5 +535,53 @@ export function visitBlock(
   });
 }
 
+/**
+ * Starting at the root, explore the children without processing until one or more start nodes are found.
+ *
+ * For each Start nodes, apply {@link visitEachChild} to it with the given callback.
+ */
+export function visitSpecificChildren<T extends FunctionlessNode>(
```

Comment:
What's a use case for this?

Reply:
```ts
new AwaitExpr(new PromiseExpr(new CallExpr()))
```

I want to explore the structure of Await and Promise, but don't start call the callback until we hit `CallExpr`.

Basically a range of nodes to take as is, without knowing their structure.